lib/zooniverse_github.rb

```diff
@@ -107,8 +107,78 @@ def update_production_migrate_tag(repo_name)
         update_tag(full_repo_name, deploy_ref)
       end
 
+      def get_dependabot_issues(last_repo_listed)
+        query = last_repo_listed.nil? ? query_without_after : query_with_after(last_repo_listed)
```

Comment:
i'd remove the nil? checks here 
```suggestion
        query = last_repo_listed ? query_with_after(last_repo_listed) : query_without_after
```